USB: Package usb-net-cdc-mbim missing dependcy on usb-net-cdc-ncm

diff --git a/package/kernel/linux/modules/usb.mk b/package/kernel/linux/modules/usb.mk
--- a/package/kernel/linux/modules/usb.mk
+++ b/package/kernel/linux/modules/usb.mk
@@ -1288,6 +1288,7 @@ define KernelPackage/usb-net-cdc-mbim
   KCONFIG:=CONFIG_USB_NET_CDC_MBIM
   FILES:= \
    $(LINUX_DIR)/drivers/$(USBNET_DIR)/cdc_mbim.ko
+  DEPENDS:= +kmod-usb-net-cdc-ncm
   AUTOLOAD:=$(call AutoProbe,cdc_mbim)
   $(call AddDepends/usb-net,+kmod-usb-wdm +kmod-usb-net-cdc-ncm)
 endef
